ScopeTest fails on Windows

When run on Windows 10:
```
19:09:22.766 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created scheduled thread pool 'file-processing' of size 5
19:09:22.794 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.internal.provider.TemplateResourceBundleProvider" has been added.
19:09:22.794 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.internal.provider.file.TemplateFileProviderWatcher" has been added.
19:09:22.817 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.module.script.internal.provider.ScriptModuleTypeProvider" has been added.
19:09:22.817 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.internal.provider.file.ModuleTypeFileProviderWatcher" has been added.
19:09:22.835 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorageService - Json Storage Service: Activated.
19:09:22.837 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\automation_rules.json' does not exist - checking for a backup.
19:09:22.843 [Automation Provider Processing Queue] DEBUG org.openhab.core.automation.internal.provider.RuleResourceBundleImporter - Parse rules from bundle 'org.openhab.core.automation.module.script.defaultscope.tests'
19:09:22.843 [Automation Provider Processing Queue] DEBUG org.openhab.core.automation.internal.provider.RuleResourceBundleImporter - Parse rules from bundle 'org.openhab.core.automation'
19:09:22.844 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.ManagedRuleProvider" has been added.
19:09:22.852 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - Added GenericScriptEngineFactory
19:09:22.853 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - ScriptEngineFactory details for Oracle Nashorn (1.8.0_131): supports ECMAScript (ECMA - 262 Edition 5.1) with file extensions [js], names [nashorn, Nashorn, js, JS, JavaScript, javascript, ECMAScript, ecmascript], and mimetypes [application/javascript, application/ecmascript, text/javascript, text/ecmascript]
19:09:22.854 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - Added NashornScriptEngineFactory
19:09:22.854 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - ScriptEngineFactory details for Oracle Nashorn (1.8.0_131): supports ECMAScript (ECMA - 262 Edition 5.1) with file extensions [js], names [nashorn, Nashorn, js, JS, JavaScript, javascript, ECMAScript, ecmascript], and mimetypes [application/javascript, application/ecmascript, text/javascript, text/ecmascript]
19:09:22.861 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\automation_rules_disabled.json' does not exist - checking for a backup.
19:09:22.862 [main] DEBUG org.openhab.core.automation.internal.RuleEngineImpl - ModuleHandlerFactory added ScriptModuleHandlerFactory
19:09:22.872 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.thing.Thing.json' does not exist - checking for a backup.
19:09:22.887 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.thing.ManagedThingProvider" has been added.
19:09:22.891 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.thing.link.ItemChannelLink.json' does not exist - checking for a backup.
19:09:22.918 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created thread pool 'safeCall' with size 5
19:09:22.961 [main] DEBUG org.eclipse.smarthome.core.internal.i18n.I18nProviderImpl - No language set, setting locale to 'null'.
19:09:22.961 [main] DEBUG org.eclipse.smarthome.core.internal.i18n.I18nProviderImpl - Error setting measurement system for value 'null'.
19:09:22.970 [main] DEBUG org.openhab.core.automation.internal.RuleEngineImpl - ModuleHandlerFactory added AnnotatedActionModuleTypeProvider
19:09:22.970 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.internal.module.provider.AnnotatedActionModuleTypeProvider" has been added.
19:09:22.972 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.internal.provider.ModuleTypeResourceBundleProvider" has been added.
19:09:22.975 [main] DEBUG org.openhab.core.automation.internal.RuleEngineImpl - ModuleHandlerFactory added AnnotatedThingActionModuleTypeProvider
19:09:22.975 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.openhab.core.automation.thingsupport.AnnotatedThingActionModuleTypeProvider" has been added.
19:09:22.984 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.items.Metadata.json' does not exist - checking for a backup.
19:09:23.001 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.internal.items.ManagedMetadataProviderImpl" has been added.
19:09:23.002 [main] DEBUG org.openhab.core.automation.internal.RuleEngineImpl - ModuleHandlerFactory added CoreModuleHandlerFactory
19:09:23.025 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.thing.link.ManagedItemChannelLinkProvider" has been added.
19:09:23.031 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created scheduled thread pool 'thingLinkManager' of size 5
19:09:23.052 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created scheduled thread pool 'scheduler' of size 5
19:09:23.054 [main] DEBUG org.openhab.core.automation.internal.RuleEngineImpl - ModuleHandlerFactory added TimerModuleHandlerFactory
19:09:23.058 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created thread pool 'ConfigStatusService' with size 5
19:09:23.071 [main] DEBUG org.eclipse.smarthome.core.thing.internal.firmware.FirmwareUpdateServiceImpl - Modifying the configuration of the firmware update service.
19:09:23.071 [main] DEBUG org.eclipse.smarthome.core.thing.internal.firmware.FirmwareUpdateServiceImpl - Given time unit null is not supported. Will keep current configuration.
19:09:23.082 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.items.Item.json' does not exist - checking for a backup.
19:09:23.083 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.items.ManagedItemProvider" has been added.
19:09:23.090 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.thing.internal.ChannelItemProvider" has been added.
19:09:23.101 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created scheduled thread pool 'thingManager' of size 5
19:09:23.102 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.automation.module.script.defaultscope.tests\target\test\tmp\testing\itest\userdata\jsondb\thing_status_storage.json' does not exist - checking for a backup.
19:09:23.499 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - Added ScriptEngine for language 'js' with identifier: myJSEngine
TEST testScopeDefinesItemTypes(org.openhab.core.automation.module.script.defaultscope.internal.ScopeTest) <<< ERROR: null
java.lang.NullPointerException
        at org.openhab.core.automation.module.script.defaultscope.internal.ScopeTest.testScopeDefinesItemTypes(ScopeTest.java:53)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
        at junit.framework.TestSuite.runTest(TestSuite.java:252)
        at junit.framework.TestSuite.run(TestSuite.java:247)
        at aQute.junit.Activator.test(Activator.java:340)
        at aQute.junit.Activator.automatic(Activator.java:236)
        at aQute.junit.Activator.run(Activator.java:177)
        at aQute.launcher.Launcher.lambda$serviceChanged$0(Launcher.java:1385)
        at aQute.launcher.Launcher.run(Launcher.java:352)
        at aQute.launcher.Launcher.main(Launcher.java:152)
19:09:23.539 [main] DEBUG org.openhab.core.automation.module.script.internal.ScriptEngineManagerImpl - Added ScriptEngine for language 'js' with identifier: myJSEngine
TEST testScopeDoesNotDefineFoobar(org.openhab.core.automation.module.script.defaultscope.internal.ScopeTest) <<< ERROR: Unexpected exception, expected<javax.script.ScriptException> but was<java.lang.NullPointerException>
java.lang.Exception: Unexpected exception, expected<javax.script.ScriptException> but was<java.lang.NullPointerException>
        at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:28)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
        at junit.framework.TestSuite.runTest(TestSuite.java:252)
        at junit.framework.TestSuite.run(TestSuite.java:247)
        at aQute.junit.Activator.test(Activator.java:340)
        at aQute.junit.Activator.automatic(Activator.java:236)
        at aQute.junit.Activator.run(Activator.java:177)
        at aQute.launcher.Launcher.lambda$serviceChanged$0(Launcher.java:1385)
        at aQute.launcher.Launcher.run(Launcher.java:352)
        at aQute.launcher.Launcher.main(Launcher.java:152)
Caused by: java.lang.NullPointerException
        at org.openhab.core.automation.module.script.defaultscope.internal.ScopeTest.testScopeDoesNotDefineFoobar(ScopeTest.java:59)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
        ... 20 more
Tests run  : 2
Passed     : 0
Errors     : 2
Failures   : 0
[INFO] 2 Error(s)
```

--- a/itests/org.openhab.core.automation.module.script.defaultscope.tests/src/main/java/org/openhab/core/automation/module/script/defaultscope/internal/ScopeTest.java
+++ b/itests/org.openhab.core.automation.module.script.defaultscope.tests/src/main/java/org/openhab/core/automation/module/script/defaultscope/internal/ScopeTest.java
@@ -12,7 +12,6 @@
  */
 package org.openhab.core.automation.module.script.defaultscope.internal;
 
-import java.io.File;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URL;
@@ -35,7 +34,7 @@ public class ScopeTest extends JavaOSGiTest {
 
     private ScriptEngine engine;
 
-    private final String path = "ESH-INF" + File.separator + "automation" + File.separator + "jsr223";
+    private final String path = "ESH-INF/automation/jsr223/";
     private final String workingFile = "scopeWorking.js";
     private final String failureFile = "scopeFailure.js";
 
@@ -48,13 +47,13 @@ public class ScopeTest extends JavaOSGiTest {
 
     @Test
     public void testScopeDefinesItemTypes() throws ScriptException, IOException {
-        URL url = bundleContext.getBundle().getResource(path + File.separator + workingFile);
+        URL url = bundleContext.getBundle().getResource(path + workingFile);
         engine.eval(new InputStreamReader(url.openStream()));
     }
 
     @Test(expected = ScriptException.class)
     public void testScopeDoesNotDefineFoobar() throws ScriptException, IOException {
-        URL url = bundleContext.getBundle().getResource(path + File.separator + failureFile);
+        URL url = bundleContext.getBundle().getResource(path + failureFile);
         engine.eval(new InputStreamReader(url.openStream()));
     }
 }

